### PR TITLE
[MODFISTO-500] - Implement endpoint to return all finance data for FY

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
https://folio-org.atlassian.net/browse/MODFISTO-500
## Approach
 - created object to handle finance data for fiscal year

### CODE REVIEW

File: mod-finance/schemas/fy_finance_data.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Finance data for a fiscal year",
+  "type": "object",
+  "properties": {
```
Comment (SerhiiNosko) [inline mod-finance/schemas/fy_finance_data.json:5]: lets also add add fiscalYearId into response

File: mod-finance/schemas/fy_finance_data.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        "Active",
+        "Frozen",
+        "Inactive"
+      ]
+    },
+    "fundTags": {
+      "type": "object",
+      "description": "Arbitrary tags associated with this fund",
+      "$ref": "../../../raml-util/schemas/tags.schema"
+    },
```
Comment (SerhiiNosko) [inline mod-finance/schemas/fy_finance_data.json:35]: also lets add fundDescription per this screen
image

File: mod-finance/schemas/fy_finance_data.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        "Inactive",
+        "Planned",
+        "Closed"
+      ]
+    },
+    "budgetInitialAllocation": {
+      "default": 0,
+      "description": "The amount of the first allocation made to this budget",
+      "type": "number"
+    },
```
Comment (SerhiiNosko) [inline mod-finance/schemas/fy_finance_data.json:59]: we also need to include field budget.allocated per this screen
image

Comment (SerhiiNosko) [reply mod-finance/schemas/fy_finance_data.json:63]: yes, we need to provide both allocation per this csv file: initialAllocation and allocated
image

Comment (azizbekxm) [reply mod-finance/schemas/fy_finance_data.json:67]: thanks to clarfiy, added allocated field as budgetCurrentAllocation

**Review by SerhiiNosko — APPROVED** [review]

**Review by BKadirkhodjaev — APPROVED** [review]

**Pull request merged**